components/grid/overview.md

```diff
@@ -175,6 +175,7 @@ The Grid has methods to execute actions such as:
 * [export to Excel]({%slug grid-export-excel%}) and other formats
 * [automatically resize columns to fit their content]({%slug components/grid/columns/resize%}#autofit-columns)
 * [get or set the Grid configuration state]({%slug grid-state%})
+* get the corresponding `TItem` of the destination Grid from the passed [`DestinationIndex`]({%slug grid-drag-drop-overview%}#event-arguments) - `GetItemFromDropIndex(string index)`
```

Comment:
This sentence is too technical and not understanble out of context. We can be more general -

```suggestion
* [get the dragged data item and its drop index from the destination Grid instance]({%slug grid-drag-drop-overview%})
```